src/routes/api/db-migrations/db-migration.test.ts

```diff
@@ -0,0 +1,116 @@
+import express from "express";
```

Comment:
Shouldn't it be called `db-migration-router.test.ts` ?

Reply:
Ah, I see it is for both up and down... Would it make sense to merge them together into one file then to keep the convention? Currently it was tricky for me to relate the tests...

Reply:
Or maybe it would make sense to split this 2 file onto 2 (for up and down)?

Reply:
Yup make sense, split into two files, one for up and one for down.
I'll pass the router test for now, as it should be covered already in each of the up/down handler test.